docs/integration/external-integrations/device42-servicenow-connector.mdx

```diff
@@ -302,11 +297,11 @@ Configure the data sync schedules as appropriate for your environment. Search Se
 
 ![](/assets/images/SN_Store_Download_4.png)
 
-These can be scheduled to run at certain times and should be run in the highlighted order: Devices, IPs, Services, Software Details (version 2.0.1), and Software Instances. You can click on an individual schedule to edit it or to **Execute Now**.
+These can be scheduled to run at certain times and should be run in the highlighted order: Affinity Group, Application Components, Business Applications, Database, Database to Instances, D42-Web-Sites, Web Site, Devices, IPs, Services, and Software Instances. You can click on an individual schedule to change running order , period and time. And then you can **Execute Now**.
```

Comment:
Replace this paragraph with:
`The Device scheduled job is set to run at a default time which can be modified if needed. The Device scheduled job will automatically trigger the next scheduled job to run, which will then trigger the next scheduled job, and so on until all of the scheduled jobs are run.  The order the scheduled jobs will run is: Device, IP, Software, Service, Affinity Group, Application Component, Business Application, Database, Database Instance, Database to Instance, Website. You can click on the Device scheduled job to change period and time. And then you can **Execute Now**.`